### ISSUE TITLE
[suggestion] Add config to adjust harvest level of living tools

### ISSUE THREAD

**Issue by Wakatsu** [Lothrazar/RootsClassic#101]
What it says on the can - either buff living pickaxe to diamond level, or add a config to let server owner decide what level it should be. Currently there is next to no incentive to craft it, since the only benefit of living toolset (aka inbuild regeneration) is only useful to have on tools that you are planning to stick with long term, as it eliminates the problem with rising upkeep. But for iron tools there is no need to preserve them since 1 - iron is almost as common as dirt and 2 - iron tools only usage is to mine 3 diamonds for the diamond pick, after which you'll never go back to iron tier again.

**Comment by BlondfootInnawoods** [comment]
I see what you're saying, though I'm not sure it should be diamond level. What you're saying is applicable to frenetic competition speedrunning but not casual "slow down and smell the roses" gameplay. Living tools are flavorful for woodsman herbalist roleplay, and not everyone goes hog wild digging straight down all the way to bedrock within a minute of punching their first tree, making and filling a water bucket, and maybe killing a few nearby pigs. Also, I can't tell you how many times I've broken diamond picks before I've min-maxed my way to a mending librarian villager. My fault for not playing optimally, but y'know I'm not trying to impress anybody by chasing world records...

**Comment by Wakatsu** [comment]
I mean, yeah, true, but I'm rising this issue precisely BECAUSE I want to be able to play a nature-themed character all the way till the end and I'm tired of trying to find mods that can do this. Instead I'm forced to switch to dark magic \ tech \ vanilla after midgame, which is sad hours :\
And please don't bring in Botania, this is a TECH mod (even by its own creator's words) and it is heavily skewed towards industrialization. And a powerplant is still a powerplant when hidden behind a flowery facade.
I LOVED living tools and armor in Roots 2 and was very sad I had to replace them due to needing diamond-tier stuff later in the game. I wanted to keep my green tool set. Back in the days Thaumcraft could be used to help with that (you could infuse your tools to raise their tier), but nowadays you have to switch eventually. No matter the speed you play - you can't live in midgame forever.

**Comment by BlondfootInnawoods** [comment]
Oh wow, I didn't think anybody would see this, let alone reply!

Yeah, I agree. It'd be cool to not have to choose between flavor and function, to accept crappy armor and tools in place of survival. Can't roleplay if you're dead. After I thought about this maybe if one could upgrade a living tool with a diamond, then upgrade that hypothetical living diamond into living netherite, that'd be cool, especially if it had subtle slow animations of writhing thorny vines and/or pulsing glowing mushrooms 'n' stuff!

And yeah true, one does need to get to getting if they don't want to keep dying to every surprise creeper, pair of skeletons you didn't have a chance to draw sword and board and charge, or going from fine to find red mist under a second because you couldn't cut down that zombie horde fast enough with even a Sharpness V Fire Aspect II netherite sword.

Sorry if I was aggressive when I went overly long about speedrunning and overblowing and mischaracterizing what you were saying.

**Comment by Wakatsu** [comment]
>>I like the idea of taking your first wooden pickaxe and upgrading it all the way to the ender dragon.

Yeah, that! Recycle\upgrade instead of making a new pick every time. People like upgradeable things so much because it allows you to build connection to your stuff. When your adventure is over, you'll be leaving behind a sword that is practically an artifact now with its own history starting as a simple sharpened twig and going all the way into netherite-infused enchanted dragon-slaying flamberge. Which also parallels your own journey.
Even mojang seem to understand that, seen as they made it a mechanic with a smithing table.

>Sorry if I was aggressive when I went overly long about speedrunning and overblowing and mischaracterizing what you were saying.

Been there, bud XD We cool)
